addons/website_slides/static/src/js/public/components/slide_quiz_finish_dialog/slide_xp_progress_bar.xml

```diff
@@ -16,6 +11,11 @@
                t-out="state.rankLowerBound"/>
         <small t-if="state.rankUpperBound" class="float-end fw-bold"
                t-att-class="{'d-none': state.hideRankBounds}" t-out="state.rankUpperBound"/>
+        <div class="tooltip fade show bs-tooltip-top position-relative progress-bar-indicator" t-att-class="{'level-up': props.levelUp}" role="tooltip"
```

Comment:
For Odoo classes, we usually prefix with "o" and the module, I see some "o_wslides" in the templates.

Maybe "o_wlides_xp_progressbar_tooltip" ? Or something like that.

Reply:
Thanks, actually, I had used that class because of the class "progress" and "progress-bar" present in that component. I have updated it with the suggested class.
Tell me if you want me to change those classes as well then.